core/input/keyboard.js

```diff
@@ -212,17 +220,17 @@ export default class Keyboard {
         if (this._altGrArmed) {
             this._altGrArmed = false;
             clearTimeout(this._altGrTimeout);
-            this._sendKeyEvent(KeyTable.XK_Control_L, "ControlLeft", true);
+            this._sendKeyEvent(KeyTable.XK_Control_L, "ControlLeft", true, null, null);
         }
 
         // See comment in _handleKeyDown()
         if ((browser.isMac() || browser.isIOS()) && (code === 'CapsLock')) {
-            this._sendKeyEvent(KeyTable.XK_Caps_Lock, 'CapsLock', true);
-            this._sendKeyEvent(KeyTable.XK_Caps_Lock, 'CapsLock', false);
+            this._sendKeyEvent(KeyTable.XK_Caps_Lock, 'CapsLock', true, null, null);
+            this._sendKeyEvent(KeyTable.XK_Caps_Lock, 'CapsLock', false, null, null);
```

Comment:
Did you consider using `undefined` instead so that callers can just omit the lock arguments?

Reply:
I can make null the default value for those parameters so that they can be omitted yes. I prefer that over using undefined directly, since it is more explicitly optional this way.